kolibri/content/migrations/0001_initial.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
```

Comment:
btw. I think we should consider banning data migrations. Have it in mind for the next Kolibri dev meeting.